lib/IR/GraphScheduler.cpp

```diff
@@ -24,15 +24,33 @@
 #include "glow/Support/Debug.h"
 
 #include "llvm/Support/Casting.h"
+#include "llvm/Support/CommandLine.h"
 #include "llvm/Support/Debug.h"
 #include "llvm/Support/raw_ostream.h"
 
+#include <queue>
+
 using namespace glow;
 
 using llvm::cast;
 using llvm::dyn_cast;
 using llvm::isa;
 
+namespace {
+llvm::cl::OptionCategory graphSchedulerCat("Graph Scheduler Options");
+
+llvm::cl::opt<SchedulerKind> graphScheduler(
+    llvm::cl::desc("Scheduler to use:"),
+    llvm::cl::values(clEnumValN(SchedulerKind::ChildMemSizeBased,
+                                "child_mem_size_based",
```

Comment:
I think that the LLVM convention is not to use underscore in flag names. For example "show-hidden" and not "show_hidden".

Reply:
Done